tests/lifecycle/lifeparameters/parameters.go

```diff
@@ -0,0 +1,27 @@
+package lifeparameters
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	WaitingTime   = 5 * time.Minute
+	RetryInterval = 5
+)
+
+var (
+	LifecycleNamespace     = "lifecycle-tests"
+	testPodLabelPrefixName = "test-network-function.com/generic"
```

Comment:
I think it would be better to change it to "test-network-function.com/lifecycle". What do you think?